lamindb/core/_settings.py

```diff
@@ -43,6 +43,36 @@ def __init__(self):
         logger.set_verbosity(self._verbosity_int)
         self._sync_git_repo: str | None = None
 
+    def __repr__(self) -> str:  # pragma: no cover
+        cls_name = colors.green(self.__class__.__name__)
+        verbosity_color = colors.yellow if self.verbosity == "warning" else colors.green
+        verbosity_str = verbosity_color(self.verbosity)
+
+        storage_root = str(self._storage_settings.root)
+        storage_str = colors.italic(storage_root)
```

Comment:
I am not sure we need colors here.

Reply:
@falexwolf what do you think?

Reply:
I think the general colors and the way I used them in this PR is pretty consistent with other models. So I'd be happy to follow up up with adapting the setup settings.

OK with you @Koncopd ?

Reply:
Yes